lesson3/Les3.py

```diff
@@ -0,0 +1,52 @@
+from bs4 import BeautifulSoup
+import requests
+import re
+import datetime
+from pymongo import MongoClient
+
+start_url = 'https://geekbrains.ru/posts'
+client = MongoClient('localhost', 27017)
+db = client.geekbrains
+collection = db.posts
+
+
+# get last page number
+resp = requests.get(start_url)
+last_page = 0
+for item in re.findall(r'/posts\?page=(\d+)', resp.text):
+    last_page = int(item) if last_page < int(item) else last_page
+
+# get all post links from all pages
+links = []
+i = 1
+while i <= 53:
```

Comment:
There should be "while i <= last_page:". I forgot to change before I created pull request.